### ISSUE TITLE
Can I host GraphiQL in .net core

### ISSUE THREAD

**Issue by pravinhabbu4u** [graphql-dotnet/graphql-dotnet#273]
Hello,

So far I was able to host GraphQL for around 30+ API's using node implementation. Now we are trying to implement the same solution using .net core. I was able to create a schema in .net core solution as well. However, I am not sure how I can expose the graphiQL end point in .net core. Can someone help me with this?

**Comment by pravinhabbu4u** [comment]
Maybe this needs to be posted under https://github.com/graphql/graphiql

**Comment by FrederickBrier** [comment]
I integrated https://github.com/JosephWoodward/graphiql-dotnet 
There is graphiql in the source in some example code in this repo. I only realized that after looking at the above repo. It was fairly straightforward to integrate. I did have some minor issues with changing the default path. Eventually it executed queries, but now my schema is not longer showing up (problem-of-the-day), so I am not sure how much help I will be beyond the above URL.
